### DISCUSSION TITLE
PSBT Finalize Input without checking Output Size

### DISCUSSION THREAD

**Discussion by ynohtna92** [paulmillr/micro-btc-signer#78] [Q&A]
In the bitcoinjs library, you are able to sign and finalize an input without it checking the output size is greater than the input size. Is this something you will support? I'd like to use this library but being able to sign and finalize a single input in a PSBT is important to the application.

https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/psbt.js#L309-L332

**Comment by paulmillr** [comment]
Ordinals got easier with 1.3.0 https://github.com/paulmillr/scure-btc-signer/releases/tag/1.3.0
